### DISCUSSION TITLE
Add `--modules-dir` to pnpm docs

### DISCUSSION THREAD

**Discussion by MattSilvaa** [pnpm/pnpm#6856] [General]
Hello! Is there any chance we can add some documentation to the pnpm docs about the `--modules-dir` flag for `pnpm install` that was added as part of this [PR](https://github.com/pnpm/pnpm/commit/de99481f31d2fbdf61f05b687667676b92e9ae70)?

**Comment by MattSilvaa** [comment]
@zkochan thoughts?

**Reply by zkochan** [reply]
It is already documented: https://pnpm.io/npmrc#modules-dir

**Reply by MattSilvaa** [reply]
Ah got it, I missed that. Should it also be documented in the section for `pnpm install` options as well? I'm migrating my repos from yarn to pnpm and missed that this was an option when running `pnpm install`.  It's document in the CLI but not in the docs AFAIK.

**Reply by zkochan** [reply]
Why would you use it as a CLI option? If you want to change the modules directory, you want to change it permanently with config change. I don't think we should duplicate settings on different pages.

**Reply by MattSilvaa** [reply]
Fair enough. For my use case, my ci build script uses a different directory to store the node_modules folder and so in yarn I would use the command `yarn --production=true --modules-folder=differentFolder/node_modules` to install dependencies
